src/app/login-signup/login/login.component.ts

```diff
@@ -1,10 +1,61 @@
-import { Component } from '@angular/core';
 
+import { Component ,OnInit} from '@angular/core';
+import { FormBuilder, FormGroup, Validators , FormControl} from '@angular/forms';
+import { ActivatedRoute,Router } from '@angular/router';
+import { ReactiveFormsModule } from '@angular/forms';
+
+// an interface that contains the properties of username and password.
+interface User {
+  username: string;
+  password: string;
+}
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss']
 })
-export class LoginComponent {
+export class LoginComponent implements OnInit{
+  loginForm!: FormGroup;
+  signupUsers: User[];
+
+  constructor(private router: Router) {
+    this.signupUsers = [];
+   }
+
+  ngOnInit():void {
+    const localData = localStorage.getItem('signupUsers');
+    if(localData != null) {
```

Comment:
Use strong type for comparision. != -> !==

Reply:
Please follow this now onwards

Reply:
Changes are done